src/types.ts

```diff
@@ -84,6 +84,7 @@ export type CertCallback = (
 export interface MandatorySamlOptions {
   cert: string | string[] | CertCallback;
   issuer: string;
+  callbackUrl: string;
```

Comment:
I noticed while writing the test that `callackUrl` was incorrectly marked as optional given that there's a runtime check to make sure it's provided:

https://github.com/node-saml/node-saml/blob/e691ccf39b9f3521268fc31b8b8f099beae4654c/src/saml.ts#L96

I acknowledge that this isn't under the original scope of the PR, I'd be happy to move it to another one if requested.